sormas-api/pom.xml

```diff
@@ -11,7 +11,17 @@
 
 	<build>
 		<finalName>${project.artifactId}</finalName>
-		<resources>
+        <plugins>
```

Comment:
Remove this. sormas-api is not suppose to use java 8, since it's also used by the android app.